packages/migrate/src/__tests__/MigrateDiff.test.ts

```diff
@@ -100,17 +53,7 @@ describe('migrate diff', () => {
       const result = await MigrateDiff.new().parse(['--to-empty', '--from-url', `file:${url}`, '--script'])
       expect(result).toMatchInlineSnapshot(``)
       expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
-      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
-        -- DropTable
-        PRAGMA foreign_keys=off;
-        DROP TABLE "Post";
-        PRAGMA foreign_keys=on;
-
-        -- DropTable
-        PRAGMA foreign_keys=off;
-        DROP TABLE "User";
-        PRAGMA foreign_keys=on;
-      `)
+      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
```

Comment:
Please replace `ctx.mocked['console.info']` with `ctx.mocked['process.stdout.write']`, so we don't lose track of the SQL script in the snapshot.